tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/CreateOrderItemSteps.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Requests;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Responses;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Data;
+using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
+using TechTalk.SpecFlow;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Steps
+{
+    [Binding]
+    internal sealed class CreateOrderItemSteps
+    {
+        private readonly Request _request;
+        private readonly Settings _settings;
+        private CreateOrderItemBaseRequest _createOrderItemRequest;
+        private CreateOrderItemResponse _createOrderItemResponse;
+
+        private readonly Dictionary<string, Func<Request, string, string, CreateOrderItemBaseRequest>> _createFunctions = new Dictionary<string, Func<Request, string, string, CreateOrderItemBaseRequest>>
```

Comment:
Very minor... To simplify the steps class further this could be moved into the class `CreateOrderItemBaseRequest` with a static create function similar to a factory.